docs/concepts.rst

```diff
@@ -1,119 +1,108 @@
+Services can be connected at the project level, which means that they're scoped to a single project and cannot be shared. Global services are easy to use and share and, therefore, we recommend you rely on them for the most part. However, when you encounter a scenario where, for example, you need to use a specific Dev Hub for one - and only one(!) - project, you can simply connect to that service by way of the ``cci service connect devhub --project`` command.
 
 Tasks and Flows
 ---------------
 
-CumulusCI uses a framework of *tasks* and *flows* to organize the automation that is available to each project.
+CumulusCI uses a framework of **tasks** and **flows** to organize the automation that is available to each project.
 
-Tasks are units of automation. A task could perform a deployment, load a dataset, retrieve material from an org, install a managed package, or undertake many other activities. CumulusCI ships with scores of tasks out of the box. 
+Tasks are units of automation. A task could perform a deployment, load a dataset, retrieve material from an org, install a managed package, or undertake many other activities. CumulusCI ships with scores of tasks out of the box.
 
-You can review the tasks available in a project by running ``cci task list``, learn more about a task and how to configure its options with ``cci task info <name>``, where ``<name>`` is the name of the task, and run a task with ``cci task run <name> --org <org>``, where ``<name>`` is the name of the task and ``<org>`` is the org you'd like to run it against. For example, the ``run_tests`` task executes Apex unit tests. If you have an org called ``dev``, you can run this task against this org with the command ``cci task run run_tests --org dev``.
+You can review the tasks available in a project by running ``cci task list``; learn more about a task and how to configure its options with ``cci task info <name>``, where ``<name>`` is the name of the task; and run a task with ``cci task run <name> --org <org>``, where ``<name>`` is the name of the task and ``<org>`` is the org you'd like to run it against. For example, the ``run_tests`` task executes Apex unit tests. If you have an org called ``dev``, you can run this task against this org with the command ``cci task run run_tests --org dev``.
 
-Many operations that you'll undertake with CumulusCI, including creating new orgs, use flows. Flows are ordered sequences of tasks (and other flows!) that produce a cohesive outcome, such as an org that's configured to suit a specific workflow. 
+Many operations that you'll undertake with CumulusCI, including creating new orgs, use flows. Flows are ordered sequences of tasks (and other flows!) that produce a cohesive outcome, such as an org that's configured to suit a specific workflow.
```

Comment:
"specific workflow"

We might need to give an example of what this means, or turn the next paragraph into a more streamlined example(?).